### ISSUE TITLE
[Feature] make updateSlug conditional

### ISSUE THREAD

**Issue by bogdanrn** [todda00/meteor-friendly-slugs#30]
I want to make the updateSlug to be conditional based on the doc that is updated.

```
Products.friendlySlugs({
  slugFrom: 'title',
  slugField: 'slug',
  distinct: true,
  updateSlug: function(doc){
    return (doc.published != true);
  },
});
```

**Comment by bogdanrn** [comment]
I made a pull requrest here #31

**Comment by todda00** [comment]
Merged #31 and released with 0.5.0
